library/ssl_msg.c

```diff
@@ -1774,9 +1769,25 @@ int mbedtls_ssl_decrypt_buf( mbedtls_ssl_context const *ssl,
         }
         else
 #endif /* MBEDTLS_SSL_PROTO_SSL3 */
-#if defined(MBEDTLS_SSL_PROTO_TLS1) || defined(MBEDTLS_SSL_PROTO_TLS1_1) || \
-        defined(MBEDTLS_SSL_PROTO_TLS1_2)
-        if( transform->minor_ver > MBEDTLS_SSL_MINOR_VERSION_0 )
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
Has this section been accidentally copied?

Reply:
No, seems to be OK, but I'll check it after the CI.